5.0/en/0x53-V53-WebRTC.md

```diff
@@ -22,17 +22,21 @@ It is important to note that these security requirements **do not apply to devel
 
 ## V53.1 TURN Server
 
-TURN (Traversal Using Relays around NAT) servers play a crucial role in WebRTC applications by facilitating peer-to-peer connections in challenging network environments. However, they can also introduce security risks if not properly configured and secured. These requirements only apply to systems that host TURN servers as part of their WebRTC infrastructure.
+TURN (Traversal Using Relays around NAT) servers play a crucial role in WebRTC applications by facilitating peer-to-peer connections in challenging network environments. However, they can also introduce security risks if not properly configured and secured.
+
+These requirements only apply to systems that host TURN servers as part of their WebRTC infrastructure.
 
 | # | Description | L1 | L2 | L3 |
 | :---: | :--- | :---: | :---: | :---: |
-| **53.1.1** | [ADDED] Verify that the TURN service allows access only to IP addresses that are not reserved for special purposes (e.g., internal networks, broadcast, loopback). Note that this applies to IPv4 and IPv6 addresses. | ✓ | ✓ | ✓ |
+| **53.1.1** | [ADDED] Verify that the TURN service only allows access to IP addresses that are not reserved for special purposes (e.g., internal networks, broadcast, loopback). Note that this applies to both IPv4 and IPv6 addresses. | ✓ | ✓ | ✓ |
 | **53.1.2** | [ADDED] Verify that the TURN service is not susceptible to resource exhaustion when legitimate users attempt to open a large number of ports on the TURN server. |  | ✓ | ✓ |
 
 ## V53.2 Media
 
 Media security is paramount in WebRTC applications, as it directly impacts the confidentiality, integrity and availability of audio and video communications. By addressing these security concerns, developers can safeguard the media streams in WebRTC applications, preventing eavesdropping, tampering, and denial-of-service attacks that could compromise user privacy and communication quality.
 
+These requirements only apply to systems that...
```

Comment:
@sandrogauci do you think we could add the circumstances here when this section will or will not be applicable?

Reply:
Ok cool, can you make changes directly to this PR or do you need to make a PR on my PR @sandrogauci ?